index.js

```diff
@@ -15,8 +15,8 @@ module.exports = function metalsmithBabel(options) {
   options = Object.assign({}, options);
   let noFilesRenamed = true;
 
-  return function metalsmithBabelPlugin(files, metalsmith) {
-    for (const originalFilename of Object.keys(files)) {
+  return function metalsmithBabelPlugin(files, metalsmith, done) {
+    Object.keys(files).forEach(originalFilename => {
       const ext = extname(originalFilename).toLowerCase();
       if (ext !== '.js' && ext !== '.mjs' && ext !== '.jsx') {
         return;
```

Comment:
> What used to be `forEach(function)` became `for...of`, making the conditional `return` to terminate the entire process rather than skipping a loop iteration.

Replace this `return` with `continue` then we fix the bug more simple way.